### DISCUSSION TITLE
Just starting a new project, is it a good idea to start with diesel-async ?

### DISCUSSION THREAD

**Discussion by dxnxk-0** [weiznich/diesel_async#62] [General]
Everything is in the title. As of March of 2023 and the 0.2 version, in your opinion, can this crate be considered to be used in production ? Thanks for your input.

**Comment by weiznich** [comment]
It's up to you to decide whether a certain crate stands up with your needs for a production ready crate. That's something that cannot be answered by other persons. The only thing that I can add here is that there is obviously (as with all open source projects) no guarantee for anything.

**Reply by dxnxk-0** [reply]
I just wanted to have the guts feeling of the maintainer regarding the maturity of the project,  of course not guarantees :) thanks anyway !
